backend/app/api/endpoints/kind/skills.py

```
@@ -786,6 +1089,29 @@ def remove_single_skill_reference(
     return result
 
 
+@router.post("/{skill_id}/update-from-git", response_model=Dict[str, Any])
+def update_skill_from_git(
+    skill_id: int,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)